core-bundle/src/Command/UserPasswordCommand.php

```diff
@@ -99,7 +99,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 'password' => $hash,
                 'locked' => 0,
                 'loginAttempts' => 0,
-                'pwChange' => $input->getOption('require-change') ? '1' : '',
+                'pwChange' => $input->getOption('require-change') ? 1 : 0,
             ],
             ['username' => $input->getArgument('username')]
```

Comment:
```suggestion
                'pwChange' => $input->getOption('require-change'),
            ],
            ['username' => $input->getArgument('username')],
            [ParameterType::STRING, ParameterType::BOOLEAN, ParameterType::INTEGER, ParameterType::BOOLEAN]
```

something like that?